piccolo_api/media/s3.py

```diff
@@ -22,7 +23,10 @@ def __init__(
         column: t.Union[Text, Varchar, Array],
         bucket_name: str,
         folder_name: str,
+        cache_max_age: t.Optional[int] = None,
+        default_acl: str = "private",
```

Comment:
Maybe we should set this to ``None`` by default?

Reply:
My reasoning for this is some S3 compatible storage providers might not support ACLs. And Amazon uses private as a default anyway if it's not explicitly set.